### PR TITLE
CASSANDRA-15707: Fix cqlsh output test

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 4.0-alpha4
+ * Fix cqlsh output test (CASSANDRA-15707)
```
Comment (michaelsembwever) [inline CHANGES.txt]: i'm not too sure we need non-user-facing entries in `CHANGES.txt`

Comment (nastra) [reply CHANGES.txt]: not sure either...I can remove those as they cause merge conflicts all the time ;)

Comment (nastra) [reply CHANGES.txt]: there are a bunch of entries in CHANGES.txt that are for test fixes, so I did the same but personally either work for me

Comment (blerer) [reply CHANGES.txt]: Normally it is only user impacting changes. Not sure why people put them there.

Comment (nastra) [reply CHANGES.txt]: makes perfect sense to mention only user-relevant things in CHANGES.txt, so I removed my changes there

**Comment by nastra** [comment]
committed as 3db7796c1b7aa0e953fab621c0334c576cc7da35

**Pull request closed without merging**